paraatm/HPMM/hpmm.py

```diff
@@ -0,0 +1,101 @@
+#! /usr/bin/python3 python3
+# -*- coding: utf-8 -*-
+
+"""
+@Author: Jiawei Chen
+@Date: 2020-05-20
+This Python script is used for HPMM (Human Performance Monitoring Module)
+The script outputs predicted loss of separation as an indicator of air traffic controller's performance
+The script can also check compliance for each command
+@Last Modified by: Jiawei Chen
+@Last Modified date: 2020-05-26
+"""
+from typing import Any, Union
+
+from jpype import *
+from array import *
+import pandas as pd
+import numpy as np
+import scipy.stats as sts
+import os
+import time
+import sys
+import datetime
+from shutil import copyfile
+from shutil import rmtree
+
+import matplotlib.pyplot as plt
+from pandas import Series, DataFrame
+from pandas.io.parsers import TextFileReader
+from scipy.stats import norm
+from sklearn.preprocessing import StandardScaler
+from sklearn.model_selection import train_test_split
+import catboost as cb
+from catboost import CatBoostRegressor
+from catboost import Pool
+
+
+
+class HPMM(object):
+    def __init__(self, cfg):
+        self.data = pd.read_csv(cfg['datafile']);
```

Comment:
Why have you chosen to use a dictionary with one key as the input here?  Why not just pass in the name of the data file as a string, or even better pass in array data structures with the data?  Having the function accept one or more array data structures will be more consistent with how other data analysis methods are designed in Python, and it avoids the need to make assumptions about the format of this data file that is being provided by the user.

Also, since this is a "public" function that users of your module would interact with, please add a docstring that includes at a minimum describes the function parameters.  See for example [iff.py](../blob/master/io/iff.py).